### ISSUE TITLE
Installer Issue

### ISSUE THREAD

**Issue by jmpartington** [JonZwe/PFAScreen#5]
Hi,

I've been following the provided instructions for PFAScreens installation and have had no luck.
I've completely uninstalled Python from my PC (Windows 10) and tried using the Installation.bat file to reinstall Python (3.12) and then complete the rest of the installation as advised.

After completing the steps for installation, when Run_PFAScreen is opened, a command window briefly appears and disappears, and then nothing happens.

I've tried the installation on a number of computers, including those where I am and aren't an admin.
I'm happy to provide some screenshots or command log if needed. Just thought I'd raise this issue to see if anyone else has experienced the same.

Thanks for your time!

**Comment by JonZwe** [comment]
Hi, 
thanks for reporting this issue. I might know what the problem is. This could happen if several Python versions are installed on your computer. To fix it you can try the following: Navigate into the PFAScreen folder and rightclick the Run_PFAScreen.bat file and go on with "edit". This should open the file in a text editor. Replace the text with "python run_PFAScreen.py" (without quotation marks) and save it. The you close the editor and try it again.
Let me know if this could fix the error!
Best regards,
Jonathan

**Comment by EllisSGMackay** [comment]
> Hi, thanks for reporting this issue. I might know what the problem is. This could happen if several Python versions are installed on your computer. To fix it you can try the following: Navigate into the PFAScreen folder and rightclick the Run_PFAScreen.bat file and go on with "edit". This should open the file in a text editor. Replace the text with "python run_PFAScreen.py" (without quotation marks) and save it. The you close the editor and try it again. Let me know if this could fix the error! Best regards, Jonathan

Hi Jonathan, 
@jmpartington and I worked on it tonight and got it to work, I have submitted a pull request with my fixes. It was an issue with the python working directories on our machines, my changes should ensure that the script and the batch file run regardless of this issue in the future without impacting anything else.
Cheers, 
Ellis

**Comment by jmpartington** [comment]
Hi Johnathan,
Also following up on @EllisSGMackay's comment. Whilst working on this issue earlier, we found that Python 3.12 is not suitable as does not meet some of the requirements of requirements.txt (for example PILLOW). Giving the following error during PFAScreen installation:

> ERROR: Ignored the following versions that require a different python version: 1.21.2 Requires-Python >=3.7,<3.11; 1.21.3 Requires-Python >=3.7,<3.11; 1.21.4 Requires-Python >=3.7,<3.11; 1.21.5 Requires-Python >=3.7,<3.11; 1.21.6 Requires-Python >=3.7,<3.11

Both @EllisSGMackay and I got PFAScreen running on Python 3.10, using the changes included in thier pull request.
Cheers,
Jordan
